### PR TITLE
add web HoF for Q1 2022

### PR DESCRIPTION
**Pull request by fkiriakos07**

### CODE REVIEW

File: bug-bounty-hof/web.yml [hunk 1]
```diff
@@ -1,4 +1,69 @@
...
+  url: https://twitter.com/@shhnjk
+- name: Codermak
+  date: 2022-03-22
+  twitter: "@arshadkazmi42"
+  url: https://twitter.com/@arshadkazmi42
+- name: zeb0x01
+  date: 2022-03-15
+  url: https://twitter.com/https://www.linkedin.com/in/mehboob-shaikh-7791a3169
+- name: Anonymous
+  date: 2022-03-15
```
Comment (fkiriakos07) [inline bug-bounty-hof/web.yml:20]: Hello @tomrittervg , how can I know who this Anonymous entry from? I checked the bugs and I didn't find this

Comment (fkiriakos07) [reply bug-bounty-hof/web.yml:20]: reporter asked to be anonymous

**Comment by tomrittervg** [comment]
Awesome, looks great.  Only saw a couple of issues:

When you have a name for a HOF entry but no bounty attachment you'll need to 

```
./update_hof.py -a ... --hmac email@email.com
```

Then take that hash and put nit manually in the correct place for the credit entries so we can handle their credit correctly in the future.  Those fields get auto-created from Bounty attachments, but for HOF only we need to make them manually. 

It looks like Khalid was actually in Q2, but we can include him as long as we de-dupe him next month.

----

I've used your patch to correct where the one I generated was wrong (and fixed those issues above.)  I'm running the client one now and will put it up for review by you and Dan tomorrow I expect.

**Comment by fkiriakos07** [comment]
Yes I had to do some manual changes to account for the missing attachments.

regarding Khalid's entry, I added the date from when the issue was resolved, would that work? would he be added to that particular quarter?

**Comment by tomrittervg** [comment]
Actually, he will become the sole entry in a new Q2 quarter, so we probably don't want to do that...

**Comment by tomrittervg** [comment]
I put up #97

**Pull request closed without merging**